websites/management/commands/update_content_fields.py

```diff
@@ -0,0 +1,63 @@
+"""Updates multiple fields of content based on starter"""
+from django.core.exceptions import FieldDoesNotExist
+from django.core.management import BaseCommand, CommandError
+from django.db import transaction
+
+from websites.models import WebsiteContent
+
+
+class Command(BaseCommand):
+
+    help = __doc__
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "starter",
+            help="The WebsiteStarter slug to process",
+        )
+
+        parser.add_argument(
+            "-t",
+            "--type",
+            dest="type",
+            default="testimonials",
+            help="The type of content to process",
+        )
+
+        parser.add_argument(
+            "-k",
+            "--keys",
+            dest="keys",
+            nargs="+",
+            default=["type", "dirpath"],
+        )
+
+        parser.add_argument(
+            "-d",
+            "--data",
+            dest="data",
+            nargs="+",
+            default=["stories", "content/stories"],
+        )
+
+    def is_valid(self, fields, values):
+        return len(fields) == len(values)
+
+    def handle(self, *args, **options):
+
+        fields = options.get("keys")
+        values = options.get("data")
+        starter = options.get("starter")
+        page_type = options.get("type")
+
+        if not self.is_valid(fields, values):
+            raise CommandError("The number of arguments for keys should match data")
+
+        updated_data = {x: y for x, y in zip(fields, values)}
+        with transaction.atomic():
+            try:
+                WebsiteContent.objects.filter(
+                    website__starter__slug=starter, type=page_type
+                ).update(**updated_data)
```

Comment:
So, `content_type` is a special field.  The value for this is actually stored in `type` on the `WebsiteContent` object, but is serialized as `content_type`.  See https://github.com/mitodl/ocw-studio/blob/4162f9b100928caaccf9ed874d82ce84ee2f0f7e/content_sync/serializers.py#L78

The reason for this is that `type` is a reserved property in Hugo that overrides the layout selection for a given piece of content: https://gohugo.io/content-management/front-matter/ (see `type` section).  In this case, the `type` of "testimonials" will need to be migrated to be `stories`.  The `dirpath` on these `WebsiteContent` objects will also need to be migrated to `content/stories`.  For example: https://ocw-studio-rc.odl.mit.edu/admin/websites/websitecontent/176179/change/?_changelist_filters=q%3Docw-www%26type%3Dtestimonials

A change will still need to be made in `ocw-hugo-themes`; the folder at `course/layouts/testimonials` will need to be renamed to `course/layouts/stories`.  This is one of those "chicken and the egg" situations though where both changes need to be released simultaneously.  If you either of the changes without the other, it will result in those pages being rendered using the default single page layout.